src/utils/options.js

```diff
@@ -192,4 +192,39 @@ export function processWrapperOptions(optionsRaw) {
     }
 
     return options;
+}
+
+/**
+ * Update rowSort information in options.
+ * @param {(object|object[]|null)} optionsRaw The raw value of the options prop.
+ * @param {object} sortInfo The name and type of data field and sorting order.
+ */
+export function updateRowSortOptions(optionsRaw, sortInfo) {
```

Comment:
Oh, is there a case where the options are stored in an array but do not have a default option? If so, this might fail, so we have to check for the `undefined` value.

I will add a test for this function!

Reply:
Right now our demo of the options array does contain a default object, but I do not think we should assume that it will always be provided

Reply:
Got it! I will add that part too.